content/docs/features/kubernetes-runtime-scans/_index.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
I'd replace the content of this file with something similar to the `docker_scanning.md` for example, as currently this still refers to the features of KubeClarity instead of the Kubernetes scanning capability of VMClarity->OpenClarity.

Reply:
S I should remove the content before the Asset Discovery section completely?

Reply:
> So I should remove the content before the Asset Discovery section completely?

yeah, basically, maybe the Perform ~~Runtime~~ Scan part could be kept as well and also added to the other providers because that seems to be generic to me...

Reply:
I've made the recommended changes